### PR TITLE
ISPN-13761 Remove the RocksDB cachestore from the server

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-13761

### CODE REVIEW

File: server/runtime/pom.xml [hunk 1]
```diff
@@ -87,10 +87,6 @@
          <groupId>org.infinispan</groupId>
          <artifactId>infinispan-cachestore-sql</artifactId>
       </dependency>
-      <dependency>
-         <groupId>org.infinispan</groupId>
-         <artifactId>infinispan-cachestore-rocksdb</artifactId>
```
[the remark this answers is not in the document]
Comment (tristantarrant) [reply server/runtime/pom.xml:92]: A much better idea

**Comment by tristantarrant** [comment]
@pruivo updated

**Comment by pruivo** [comment]
merged! thanks @tristantarrant !

File: documentation/src/main/asciidoc/topics/upgrading.adoc [hunk 2]
```diff
@@ -49,6 +49,14 @@ The same configuration should be changed to the following instead.
 </persistence>
 ----
 
+== RocksDB store
+
+The RocksDB implementation is no longer included by default with the server. It can be installed via the CLI:
```
Comment (wburns) [inline documentation/src/main/asciidoc/topics/upgrading.adoc:54]: Should this had said RocksDB JNI implementation?

Comment (pruivo) [reply documentation/src/main/asciidoc/topics/upgrading.adoc:54]: AFAIK, there isn't another one: https://repo1.maven.org/maven2/org/rocksdb/

**Pull request merged**